provide hyperlinks in inputInfo field (#2445)
kyma-project/busola: docs/extensibility/form-section.md

- **enum[]** - an array of options to generate an input field with a dropdown. Optionally can be a string containing a [JSONata](jsonata.md) expression returning an array of options.
- **placeholder** - specifies a short hint about the input field value.
- **required** - a boolean which specifies if a field is required. The default value is taken from CustomResourceDefintion (CRD); if it doesn't exist in the CRD, then it defaults to `false`.

> - **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.

Reviewer: ```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```

#### Widget-specific parameters

- **language** - a [JSONata](jsonata.md) expression resolving the desired language. It has access to the `$root` variable, containing the entire resource.

> - **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.

Reviewer: ```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```

- **filter** - JSONata expression that filters resources based on a given condition.
- **provideVar** - when this field is defined, the chosen resource is provided as a variable of this name.
- **required** - a boolean which specifies if a field is required. The default value is taken from CRD; if it doesn't exist in CRD, then it defaults to `false`.

> - **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.

Reviewer: ```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```

  - **name** - an optional name for a checkbox instead of the default capitalized last part of the path. This can be a key from the **translation** section.
  - **description** - a string displayed in a tooltip when you hover over a question mark icon, next to the checkbox's label.
- **required** - a boolean which specifies if a field is required. The default value is taken from CustomResourceDefintion (CRD); if it doesn't exist in the CRD, then it defaults to `false`.

> - **inputInfo** - a string below the input field that shows how to write in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.

Reviewer: ```suggestion
- **inputInfo** - a string below the input field that shows how to write in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.

  - **type** - a string that specifies the type of the value input. The options are `object`, `number`, `text`. Defaults to `text`.
  - **keyEnum[]** - an array of options to generate a key input field with a dropdown only if the `type` is set to `object`.
  - **valueEnum[]** - an array of options to generate a value input field with a dropdown.

> - **inputInfo** - a string below the input field that shows how to fill in the input. Links can be provided, {{ [`name`] (`link`) }} and optionally, a `name` instead of the `link` can be displayed.

Reviewer: ```suggestion
- **inputInfo** - a string below the input field that shows how to fill in the input. You can use the {{ [`name`] (`link`) }} format to display a `name` instead of a `link.
```
